ui/src/dashboards/actions/cellEditorOverlay.ts

```diff
@@ -8,6 +8,7 @@ import {
   ThresholdType,
   TableOptions,
 } from 'src/types/dashboards'
+import {NewDefaultCell} from '../constants'
```

Comment:
Let's pull the `NewDefaultCell` into the dashboard types instead of exporting it from the `constants` directory